File: my_robot_tutorials/scripts/robot_news_radio_transmitter.py
#!/usr/bin/env python

import rospy
from std_msgs.msg import String

if __name__ == '__main__':
    rospy.init_node('robot_news_radio_transmitter') # , anonymous = True

    pub = rospy.Publisher("/robot_news_radio", String, queue_size=10) # topic name, message type

    rate = rospy.Rate(2) # 2 messages per second

    while not rospy.is_shutdown():
        msg = String()
        msg.data = "This is robot news radio message"
        pub.publish(msg)
        rate.sleep()

rospy.loginfo("Node stopped")

File: my_robot_tutorials/src/robot_news_radio_transmitter.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>

int main (int argc, char **argv) {
    ros::init(argc, argv, "robot_news_radio_transmitter"); // , ros::init_options::AnonymousName
    ros::NodeHandle nh;

    ros::Publisher pub = nh.advertise<std_msgs::String>("/robot_news_radio", 10); // name, queue size

    ros::Rate rate(3);

    while (ros::ok()) {
        std_msgs::String msg;
        msg.data = "This is message from robot news radio.";
        pub.publish(msg);
        rate.sleep();
    }
}